robotiq_2_finger_gripper_driver/src/robotiq_2_finger_gripper_driver_node.ros2.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <memory>
+
+#include <robotiq_2_finger_gripper_driver/robotiq_2_finger_gripper_driver.hpp>
+// ROS
+#include <rclcpp/rclcpp.hpp>
+#include <robotiq_2_finger_gripper_driver/msg/robotiq2_finger_command.hpp>
+#include <robotiq_2_finger_gripper_driver/msg/robotiq2_finger_state.hpp>
+
+namespace robotiq_2_finger_gripper_driver
+{
+class Robotiq2FingerDriverNode : public rclcpp::Node
+{
+private:
+  using Robotiq2FingerState =
```

Comment:
I think it would be clearer to separate these typdefs from the private member declarations (they could be moved out of the class definition entirely, or just
```
typedefs
<empty line>
declarations
```

Reply:
Done in https://github.com/ToyotaResearchInstitute/tri_hardware_drivers/pull/13/commits/c28af2b2e048bef3a9f4cb685905a726e4a5b54e.